app/controllers/api/v2/blocks_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Api::V2::BlocksController < ApplicationController
+  # get blocks info list and paginate it.
+  #
+  # params
+  # {
+  #   "numberFrom": "10" or "0xa", //  number or integer
+  #   "numberTo": "20", // number or integer
+  #   "transactionFrom": "min transaction count", // integer
+  #   "transactionTo": "max transaction count", // integer
+  #   "page": "1", // default 1
+  #   "perPage": "10", // default 10
+  #
+  #   # offset and limit has lower priority than page and perPage
+  #   "offset": "1", // database offset for pagination
+  #   "limit": "10", //database limit for pagination
+  # }
+  #
+  # GET /v2/api/blocks
+  def index
+    options = {
+      block_number_gteq: parse_hex(params[:numberFrom]),
```

Comment:
We'd prefer `snake_case` parameters.

Reply:
No both accept `camelCase` and `snake_case`, and use `snake_case` inside.